src/components/BottomSheet/BottomSheet.js

```diff
@@ -13,34 +14,37 @@ const BottomSheet = React.forwardRef((props, ref) => {
       onOpen={() => (props.onOpen ? props.onOpen() : {})}
       closeOnDragDown={true}
       closeOnPressMask={props.closeOnPressMask}
-      height={props.height ? props.height : 260}
+      height={props.height ? props.height : 355}
       customStyles={{
         container: styles.containerSheet(pullBottom),
-        draggableIcon: styles.draggableIcon,
+        draggableIcon: styles.draggableIcon
       }}>
-      <View style={{...styles.container, ...props.viewstyle}}>
-        {props.children}
-      </View>
+      <View style={{...styles.container, ...props.viewstyle}}>{props.children}</View>
     </RBSheet>
   );
 });
 
-BottomSheet.displayName = 'BottomSheet'
+BottomSheet.displayName = 'BottomSheet';
+
+BottomSheet.propTypes = {
+  height: PropTypes.number,
+  viewstyle: PropTypes.object
+};
 
 export default BottomSheet;
 
 const styles = StyleSheet.create({
   container: {
-    paddingHorizontal: 20,
-    paddingTop: 20,
-    paddingBottom: 38,
+    paddingTop: dimen.normalizeDimen(10),
+    paddingHorizontal: dimen.normalizeDimen(20),
+    paddingBottom: dimen.normalizeDimen(20)
   },
   containerSheet: (pullBottom) => ({
     borderTopRightRadius: 20,
     borderTopLeftRadius: 20,
-    justifyContent: pullBottom ? 'flex-end' : 'flex-start',
+    justifyContent: pullBottom ? 'flex-end' : 'flex-start'
   }),
   draggableIcon: {
-    backgroundColor: colors.alto,
-  },
+    backgroundColor: colors.alto
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in the styles are generally good, as they make use of the `dimen.normalizeDimen` function to normalize dimensions across different screen sizes. However, the removal of the trailing comma in the `draggableIcon` style is inconsistent with the rest of the codebase which uses trailing commas. It's recommended to keep the code style consistent.

```diff
   draggableIcon: {
    backgroundColor: colors.alto
- 49:   }
+ 49:   },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->